src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -358,9 +388,86 @@ export const MetadataInstanceEditorAIEnabled: StoryObj<typeof MetadataSidebarRed
         expect(autofillWithBoxAI).toHaveLength(2);
 
         const editButton = await canvas.findByRole('button', { name: 'Edit My Template' });
-        userEvent.click(editButton);
+        await userEvent.click(editButton);
 
         const autofillButton = await canvas.findByRole('button', { name: 'Autofill' });
         expect(autofillButton).toBeInTheDocument();
     },
 };
+
+export const ShowErrorWhenAIAPIIsUnavailable: StoryObj<typeof MetadataSidebarRedesign> = {
+    args: {
+        features: {
+            ...mockFeatures,
+            'metadata.aiSuggestions.enabled': true,
+        },
+    },
+    parameters: {
+        ...defaultVisualConfig.parameters,
+        msw: {
+            handlers: [
+                ...defaultMockHandlers,
+                http.post('https://api.box.com/2.0/ai/extract_structured', () => {
```

Comment:
we should use the variable for our host here, in case if does change

Reply:
changed that to variable